apps/studio/components/interfaces/Integrations/Wrappers/WrappersTab.tsx

```diff
@@ -0,0 +1,104 @@
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+import { useState } from 'react'
+
+import { useParams } from 'common'
+import DeleteWrapperModal from 'components/interfaces/Database/Wrappers/DeleteWrapperModal'
+import { wrapperMetaComparator } from 'components/interfaces/Database/Wrappers/Wrappers.utils'
+import { useProjectContext } from 'components/layouts/ProjectLayout/ProjectContext'
+import { ButtonTooltip } from 'components/ui/ButtonTooltip'
+import { FDW, useFDWsQuery } from 'data/fdw/fdws-query'
+import { useCheckPermissions } from 'hooks/misc/useCheckPermissions'
+import { Sheet, SheetContent } from 'ui'
+import { INTEGRATIONS } from '../Landing/Integrations.constants'
+import { CreateWrapperSheet } from './CreateWrapperSheet'
+import { WrapperTable } from './WrapperTable'
+
+export const WrappersTab = () => {
+  const { id } = useParams()
+  const { project } = useProjectContext()
+  const [selectedWrapperForDelete, setSelectedWrapperForDelete] = useState<FDW | null>(null)
+  const [createWrapperShown, setCreateWrapperShown] = useState(false)
+  const [isClosingCreateWrapper, setisClosingCreateWrapper] = useState(false)
+  const canCreateWrapper = useCheckPermissions(PermissionAction.TENANT_SQL_ADMIN_WRITE, 'wrappers')
+
+  const { data, isLoading } = useFDWsQuery({
+    projectRef: project?.ref,
+    connectionString: project?.connectionString,
+  })
+
+  const wrappers = data?.result || []
+
+  if (isLoading) {
+    return <div>Loading</div>
```

Comment:
Does this loading state get shown? Should we make it a bit nicer?

Reply:
actually good point - this never gets called cause the loading state is handled by the parent already, will remove